solutions/USACO jan16-fortmoo.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: usaco-600
+source: USACO Platinum 2016 January
+title: Fort Moo
+author: Kevin Sheng
+---
+
+[Official Editorial](http://www.usaco.org/current/data/sol_fortmoo_platinum_jan16.html)
+
+We will use an adaption of 2D Kadane's algorithm to solve this problem.
+
+Iterate through all possible pairs of columns. For each pair, we'll solve the problem with the restriction that the fort must be aligned along these two columns. For each row between these two columns, we will use prefix sums to check if the area between them is clear.
```

Comment:
> the fort must be aligned...

Do you mean that the fort's sides must contain those columns? "Aligned" is a bit ambiguous

> For each row ... check if the area between them is clear

What is "them"?

Reply:
1. If I say that if the fort contained those two columns, people might misinterpret it as being if the columns were right inside the fort, not contained in the walls.
2. "them" is the area in the row between the 2 columns.